src/main/java/ua/softserve/rv_028/issuecitymonitor/DBSeeder.java

```diff
@@ -39,18 +36,23 @@ public DBSeeder(EntityManagerFactory factory, BCryptPasswordEncoder encoder){
             fillDatabase();
             LOGGER.info("Seeding finished");
         } catch (RuntimeException e) {
-            LOGGER.error("Seeding has been done already. Skipping...");
+            LOGGER.info("Seeding has been done already. Skipping...");
         }
     }
 
     private void fillDatabase() throws RuntimeException {
         Session session = sessionFactory.openSession();
         Transaction transaction = session.beginTransaction();
 
+        User defaultAdmin = new User("Root", "Admin", encoder.encode("root"), "admin@mail.com", "+380997755331",
```

Comment:
Please change default admin password to `1234`.

Reply:
Nope, i like root password